docs/miniCore.md

```diff
@@ -0,0 +1,194 @@
+# A Minimal Core and Clearly Defined Extension Points
```

Comment:
Please add author name and date.

Then "I" makes sense!

Reply:
It's still written as an email I fear. However, if only I knew how to modify my own pull request...

Reply:
Edit your copy and add commits to the branch [rat10:patch-6](https://github.com/rat10/rdf-star-wg/tree/patch-6).

Then either rebase locally to clean up, or we can squash to one commit at the point of merge.

Reply:
Thanks. I just saw that TallTed already provided a patch.